src/js/start.js

```diff
@@ -85,7 +85,9 @@ $(function () {
     $(this).attr('tabindex', '-1');
   });
 
-  toggleFormInput($('.usa-show_password'), 'Show Password', 'Hide Password');
-  toggleFormInput($('.usa-show_multipassword'), 'Show my typing', 'Hide my typing');
-  validator($('.js-validate_password'));
+  if (/form-controls/.test(window.location.pathname)) {
```

Comment:
Just realized something about this:

The code compiled from `start.js` is distributed as part of the WDS separate from the docs site, but this code is assuming you're on the docs site because it's testing for URLs of the docs site. So if a user had the standards on their own site, this if statement wouldn't be called correctly because they won't have a `form-controls` page (it's really confusing knowing how the WDS is separate from the docs site)

What is this check attempting to protect from? Would it make sense to uses classes rather then URL's to test for existence?